server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/TestCicsWaitCics.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2024 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.usecases;
+
+import com.google.common.collect.ImmutableMap;
+import org.eclipse.lsp.cobol.common.error.ErrorSource;
+import org.eclipse.lsp.cobol.usecases.common.CICSTestUtils;
+import org.eclipse.lsp4j.Diagnostic;
+import org.eclipse.lsp4j.DiagnosticSeverity;
+import org.eclipse.lsp4j.Range;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+/**
+ * Test CICS WAITCICS command. Documentation link: <a
+ * href="https://www.ibm.com/docs/en/cics-ts/6.x?topic=summary-waitcics">WAITCICS
+ * Command</a>
+ *
+ * <p>This class tests all variations of the WAITCICS command found in the link above.
+ */
+public class TestCicsWaitCics {
+    private static final String WAITCICS_VALID_ALL_OPTIONS_ONE = "WAITCICS ECBLIST(1) NUMEVENTS(2) NAME(3) PURGEABLE";
+    private static final String WAITCICS_VALID_ALL_OPTIONS_TWO = "WAITCICS ECBLIST(1) NUMEVENTS(2) PURGEABILITY(4) NAME(3)";
+    private static final String WAITCICS_VALID_ALL_OPTIONS_THREE = "WAITCICS NUMEVENTS(2) ECBLIST(1) NAME(3) NOTPURGEABLE";
+
+    private static final String WAITCICS_VALID_SOME_OPTIONS_ONE = "WAITCICS NAME(3) ECBLIST(1) NUMEVENTS(2)";
```

Comment:
I am getting the follow error:
```
 LINE           SOURCE LISTING                                                                                      
                                                                                                                    
00001                     EXEC CICS WAITCICS NAME(3) ECBLIST(1) NUMEVENTS(2)       00079406                         
00002                                                                              00079304                         
00003                     END-EXEC                                                 00079506                         
00004                                                                              00079604                         
CICS 7.3.0 COMMAND LANGUAGE TRANSLATOR                                            TIME 04.08 DATE 27 NOV 24   PAGE 2
TRANSLATOR DIAGNOSTIC MESSAGES OF SEVERITY W AND ABOVE.                                                             
MESSAGE NUMBER  SEVERITY  LINE NUMBER     MESSAGE                                                                   
DFH7036I           W        00002         PROGRAM DOES NOT BEGIN WITH AN IDENTIFICATION DIVISION.                   
DFH7215I           W        00002         SUB-OPERAND '3' OF 'NAME' KEYWORD SHOULD BE DATATYPE CHARACTER.           
DFH7259I           E        00002         OPERAND '1' OF KEYWORD 'ECBLIST' IS INVALID FOR COBOL. A DATA-AREA RATHER 
                                          THAN A DATA-VALUE MUST BE SPECIFIED. COMMAND NOT TRANSLATED.              
END OF TRANSLATOR DIAGNOSTIC MESSAGES.                                                                              
TRANSLATION TIME:-   0.00 MINS.                                                                                     
```
My suggestion is to just switch to variable names instead of numbers as operands.